components/resolved_ts/src/endpoint.rs

```diff
@@ -419,10 +420,18 @@ where
                     }
                 } else {
                     // follower safe-ts
+                    RTS_MIN_FOLLOWER_SAFE_TS_GAP_HISTOGRAM.observe(now.saturating_sub(
```

Comment:
I don't quite understand the meaning of using `now()` to subtract the temporary variable used in the loop to calculate the minumum resolved-ts or safe-ts.

If the target is to observe the resolved-ts lag for each region, should the
```
let lag_for_region = now - read_progress_of_current_region.current_resolved_ts
histogram.observe(lag_for_region)
```
be used instead?

Reply:
yeh, this is wrong. let me fix it.